src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -250,6 +250,13 @@ module sonic-device_metadata {
                                  created, regardless of the CONFIG_DB buffers configuration.";
                 }
 
+                leaf supporting_bulk_counter_groups {
```

Comment:
We should use leaf-list for list

Reply:
I do not find a way to do it because flex counter table instances are defined differently.
See https://github.com/sonic-net/sonic-buildimage/pull/20548#discussion_r1837379871

Reply:
But you are not using leafref in current design

Reply:
Updated as the following.
each leaf represents an instance in flex counter db, eg. `PG_WATERMARK`, `QUEUE_WATERMARK`
Can you check it?

```
                leaf-list supporting_bulk_counter_groups {
                    type string;
                    description "This field contains a list of counter groups that support bulk operation.";
                }
```